storage/simulation/cluster.go

```diff
@@ -223,45 +223,90 @@ func (c *Cluster) prepareActions() {
 
 // performActions performs a single action, if required, for each range.
 func (c *Cluster) performActions() {
-	// TODO(Bram): convert this to run a single action per store instead of a
-	// single action per range.
-	for _, rangeID := range c.rangeIDs {
-		r := c.ranges[rangeID]
-		nextAction, rebalance := r.getNextAction()
-		switch nextAction {
-		case storage.AllocatorAdd:
-			newStoreID, err := r.getAllocateTarget()
-			if err != nil {
-				fmt.Fprintf(c.actionWriter, "Error: %s\n", err)
-				continue
+	tw := tabwriter.NewWriter(c.actionWriter, 12, 1, 2, ' ', 0)
```

Comment:
c.actionWriter is being used inconsistently. In this method, you do all your writing to it through this tabwriter, but elsewhere you write to it directly. At first, I thought this was intentional, because other functions do not write tabular output, but only raw "ERROR: something" strings; however, this function also outputs errors to `c.actionWriter` through this tabwriter.

I would suggest either:
- always using the tabwriter, and putting the tabwriter in the cluster struct (if the output this produces is acceptable), or
- writing errors to the raw `c.actionWriter`. In this case, you should still only allocate the tabWriter once and pass it into this function.